libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1836,19 +1834,11 @@ string YulUtilFunctions::copyArrayToStorageFunction(ArrayType const& _fromType,
 						let <elementValues> := srcPtr
 					</fromStorage>
 
-					<updateStorageValue>(elementSlot, elementOffset, <elementValues>)
+					<updateStorageValue>(elementSlot, <elementValues>)
```

Comment:
Looks like the name came from there. I'd still rename it though.
```suggestion
					<updateStorageValue>(elementSlot, <elementStackItems>)
```